index.html

```diff
@@ -7,6 +7,6 @@
   </head>
   <body>
     <h1>안녕 친구들</h1>
-    <p>반가워 여기는 새로운 우리의 세상이야.!!!</p>
+    <h1>안녕 못해 바부얌</h1>
```

Comment:
h2태그로 변경해주세요. h1 태그는 페이지당 하나가 적당합니다. 
```suggestion
    <h2>안녕 못해 바부얌</h2>
```

Reply:
어허